test/unit/inflate/Inflator.t.sol

```diff
@@ -207,8 +210,8 @@ contract InflatorTest is SharedSetup {
         targets[1] = address(counter);
 
         uint256[] memory values = new uint256[](2);
-        values[0] = 0;
-        values[1] = 0;
+        values[0] = 1 ether;
+        values[1] = 2 ether;
```

Comment:
But it doesn't test that the value was received, right? I think we should test for that.